floodviz/static/data/cities.json

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
This should not be committed. Is this the geojson that Lindsay gave us? This should be in instance/config.py. Could be represented with a single string variable that contains the json string. 

All config variables that we define should have defaults (the empty string for this one would be good) and a comment describing what you expect.